src/geom/cell_polyhedron.C

```diff
@@ -116,14 +116,23 @@ Polyhedron::Polyhedron (const std::vector<std::shared_ptr<Polygon>> & sides,
         const Polygon & side = *sides[s];
         const Point x_i = side.point(0);
         const Point n_i =
-          (side.point(1) - side.point(0)).cross
+          -(side.point(1) - side.point(0)).cross
```

Comment:
@roystgnr can I please get a fact check on this line (rest is WIP)?
I'm assuming that if the nodes are ordered in the counter-clockwise direction, the normal should point up. Is that correct?

Then placing my index along P0P1 and the major following Pn-1P0 (what this formula says?) I would get down, so instead I do
P1P0 x Pn-1P0 or similarly Pn-1P0 x P0P1 and I get up (the desired result)

am I getting this wrong?